modules/standard/Map.chpl

```diff
@@ -250,6 +250,15 @@ module Map {
       }
     }
 
+    pragma "no doc"
+    proc update(pragma "intent ref maybe const formal"
+                m: map(keyType, valType, parSafe)) {
+      compilerWarning('The `update()` method has new functionality in ' +
+                      'the 1.23 release - for forwards compatability ' +
```

Comment:
I think the message shouldn't call out a version number, it should say that this is deprecated, and finally it should mention that that this is `map.update` and not some other `update`

Maybe something like "extending a map using `update` is deprecated` - use `extend()` instead" ?

Reply:
Awesome, thanks!